data/sources/categories.json

```diff
@@ -29,55 +29,134 @@
       "excludes": ["chalice"],
       "items": [
         "3580904580",
-        "Bad Juju"
+        "Bad Juju",
+        "X Marks the Spot",
```

Comment:
can probably remove this as it is the only emote listed manually

Reply:
not understanding if there's a reason to, but i wouldn't miss it